src/Fixer/Basic/InvisibleSymbolsFixer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Basic;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Fixer Zero-width space (ZWSP), Non-breaking space (NBSP) and other unicode symbols.
+ *
+ * @author Ivan Boprzenkov <ivan.borzenkov@gmail.com>
+ */
+final class InvisibleSymbolsFixer extends AbstractFixer
```

Comment:
This rule may change content of vars, thus it's risky (I agree that in most cases it would be a proper fix, but sadly it may be desired to have it in string values in some edge cases).
For that, please mark rule as risky